api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -231,11 +238,26 @@ private User initializeUserIfNeeded() {
     // not accepting the terms of use.
 
     User user = initializeUserIfNeeded();
+    String blockscoreStatus = "not-verified";
```

Comment:
Is it necessary for us to retrieve something from BlockScore on each call to /me? Could we just get the status from them once and cache it in our DB along with the blockscore ID?

Whenever we get it from them, we need to gracefully handle the case where their server isn't responding.

Reply:
Similarly: do we want to treat id verification as an ongoing activity that happens every time the `/me` endpoint is called? OR a one-time step where we save/store the result on an account? Naively I would have thought of this as a one-time step at account creation. (In practice we could lazily wait until  `/me` is called before we check this -- but that's a different issue.)

Reply:
Fixed. It is indeed a one-time step.